product_pricelist_auto_mailing/models/product_pricelist.py

```diff
@@ -0,0 +1,23 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, _, fields
+
+
+class ProductPricelist(models.Model):
+    _name = 'product.pricelist'
+    _inherit = ['product.pricelist', 'mail.thread', 'mail.activity.mixin']
+
+    mailable_user_ids = fields.Many2many('res.users')
```

Comment:
This field is a technical one, so it should be protected, and it shouldn't be copied:
```suggestion
    mailable_user_ids = fields.Many2many('res.users', readonly=True, copy=False)
```